scripts/upgrade

```diff
@@ -18,9 +18,8 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
-is_public=$(ynh_app_setting_get --app=$app --key=is_public)
 final_path=$(ynh_app_setting_get --app=$app --key=final_path)
-lang=$(ynh_app_setting_get --app=$app --key=lang)
+language=$(ynh_app_setting_get --app=$app --key=language)
```

Comment:
You need to manage the var name change...
If not upgrade from previous version will fail